test/livebook/remote_intellisense_test.exs

```diff
@@ -89,5 +89,25 @@ defmodule Livebook.RemoteIntellisenseTest do
                insert_text: "hello($0)"
              } in Intellisense.get_completion_items("RemoteModule.hel", context, node)
     end
+
+    test "find modules from apps", %{node: node} do
+      context = eval(do: nil)
+
+      assert %{
+               label: "all_keys/1",
+               kind: :function,
+               detail: ":mnesia.all_keys/1",
+               documentation: """
```

Comment:
Don’t match on the documentation, as if Erlang update the docs, our tests break. Perhaps match only on The first sentence.